Ihor/VanGoux/scss/style.scss

```diff
@@ -27,98 +101,175 @@ body {
     padding-top: 30px;
     margin-bottom: 150px;
 
-    .header_logo {
+    &_logo {
+        min-width: 160px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
         margin-right: 200px;
+        a{
+            font-size: 20px;
+            font-weight: 600;
+            line-height: 1.5;
+            color:$blue;
+            text-decoration: none;
+        }
     }
     .header_menu{
         display: flex;
         justify-content: space-between;
-    }
+    
         .navbar {
-            min-width: 560px;
             display: flex;
             justify-content: space-between;
             align-items: center;
-            a:last-child{
-                margin-right: 40px;
-            }
-        }
-
             a {
                 @include nav_link;
+                margin-right: 60px;
             }
+        }
 
         .upload {
             @include btn;
-        }
-            .btn_text{
-                padding:10px 45px;
-            }
+            padding:10px 45px;
+        }       
+    }
 }
+@media(max-width: 640px){
+    .header_info {
+        margin-bottom: 80px;
+        padding-left: 30px;
+        .header_logo {
+            padding-top: 40px;
+            padding-left: 30px;
+        }
+        .header_menu{
+            display: none;
+        }
+    }          
+} 
 
 // promo
-.promo {
-    margin-bottom: 154px;
-}
-.promo_info{
-    h1 {
-        @include h1_promo;
-        margin-bottom: 20px;
-        margin-right: 520px;
-        
+.promo_block {
+    .promo_visit{
+        display: none;
     }
-    h5 {
-        @include h5;
-        margin-bottom: 40px;
-        margin-right: 520px;
 
-    }
-    .promo_btn{
-        @include btn;
-    }
-        .btn_text{
-            padding:10px 45px;
+    .promo {
+        margin-bottom: 154px;
+        .promo_info{
+            max-width: 600px;
+            .promo_main {
+                margin-bottom: 20px;
+                &_text {
+                    font-size: 20px;
+                    font-weight: normal;
+                    line-height: 2;
+                    color: $dark_blue;
+                    margin-bottom: 40px;
+                }
+            }
+            
+            .promo_btn{
+                @include btn;
+                padding:15px 60px;
+            }
+
+            .illustration{
+                position: absolute;
+                top: 46px;
+                right: 0;
+            }
+            margin-bottom: 160px;
         }
 
-    .illustration{
-        position: absolute;
-        top: 46px;
-        right: 0;
+        .promo_cards {
+            display: flex;
+            justify-content: space-between;
+            .card:first-child {
+                margin-right: 40px;
+            }
+            .card {
+            @include promocard;
+            }
+        }
     }
-    margin-bottom: 160px;
-}
-.promo_visit{
-    display: none;
 }
+@media(max-width: 640px){
```

Comment:
Try not to wrap long lists of styles in media, better to use media inside of each style.